ddcz/models/used/users.py

```diff
@@ -287,3 +290,27 @@ class UzivateleCekajici(models.Model):
     class Meta:
         db_table = "uzivatele_cekajici"
         unique_together = (("name_given", "name_family"),)
+
```

Comment:
I think the model should get some migrations :) 

* `UzivateleCekajici` -> `AwaitingRegistrations`
* `date` -> `timestamp`
* `supporters` -> `supporter` (I hope) 
* `patron` -> `patron_id`